### ISSUE TITLE
8BitDo Ultimate - 2.4GHz receiver disconnects loading certain cores and loading an ini with an error

### ISSUE THREAD

**Issue by massiveinvisible** [MiSTer-devel/Main_MiSTer#739]
This is an issue with some JT cores (examples are any CPS1/2 core) and the 8bitdo Ultimate controller and the 2.4GHz receiver for it. The receiver disconnects when I load a core - see attached video of this happening.
This happens exclusively with cores which show the "Assembling ROM" screen. It doesn't happen with the NeoGeo core which has a similar ROM loading screen.  Two other users in the MiSTer discord were able to replicate this issue with this controller/receiver.

The second situation where I can trigger the same issue is if I switch to an ini with an error - switching to that ini and then when the "INI error" screen appears, the receiver loses power/disconnects for a second. This doesn't happen switching to a working ini file.

https://user-images.githubusercontent.com/83599516/217657536-92679feb-cd54-4939-870f-eb6ed92b9929.mov

This happens only with the 2.4GHz receiver, it doesn't happen in bluetooth. I've tried all versions of controller and receiver firmware, my power supply is this one (https://misterfpga.co.uk/product/mister-fpga-power-supply/) and I've tried connecting the receiver through OTG. I've also tried a fresh Mr Fusion install. None of these fixed the issue.

**Comment by birdybro** [comment]
Have you tried using different USB slots?

**Comment by massiveinvisible** [comment]
Yes. All USB port on the board and I’ve tried a separate OTG hub connected to the DE10

**Comment by birdybro** [comment]
This only happens in cores made by jotego? I see you opened an issue already in jtbin:

https://github.com/jotego/jtbin/issues/313

I have to agree with his assessment, there may be some random interference and the usb dongle might get messed up temporarily. After the assembling of the rom, does unplugging and plugging the usb back in do anything?

Sounds almost like a marginal timing issue...

**Comment by massiveinvisible** [comment]
I've now tested this with the other version of the Ultimate controller which is 2.4GHz only. Arcade cores no longer cause a disconnect, however loading an ini with an error does still cause a disconnect with this controller.

**Comment by sorgelig** [comment]
That's some voodoo magic. I have no idea how MiSTer can affect 8bitdo receiver. There is no communication between MiSTer and 8bitdo radio. Looks like some bug in 8bitdo receiver. May be it simply doesn't get enough power from USB.

**Comment by Toryalai1** [comment]
Try some holy water/zam zam water on it
